### DISCUSSION TITLE
Monai label installation error

### DISCUSSION THREAD

**Discussion by mohamedtolba97** [Project-MONAI/MONAILabel#1619] [Q&A]
I am trying to install Monailabel to use it but I couldn't I always stuck at this point. what shall I do?

![Capture](https://github.com/Project-MONAI/MONAILabel/assets/156066779/4e26b1a5-8936-460d-a968-e57ef85b0c8f)

**Comment by diazandr3s** [comment]
Hi @mohamedtolba97,

This is a bit strange. First time I've seen this. Have you tried deactivating the antivirus before installing MONAI Label?
Otherwise, I'd suggest you try the Docker container: https://github.com/Project-MONAI/MONAILabel#development-version 

Let us now,
